extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -789,7 +1023,16 @@ class MultipleBackendTask extends ReplicateObject {
         return async.waterfall([
             next => this._setupRoles(sourceEntry, log, next),
             (sourceRole, next) =>
-                this._refreshSourceEntry(sourceEntry, log, next),
+                this._refreshSourceEntry(sourceEntry, log, (err, res) => {
+                    if (err && err.code === 'ObjNotFound' &&
+                        sourceEntry.getReplicationIsNFS() &&
+                        !sourceEntry.getIsDeleteMarker()) {
+                        // The object was deleted before entry is processed, we
+                        // can safely skip this entry.
+                        return next(errors.InvalidObjectState);
+                    }
```

Comment:
i think we might need to catch a general err here as well

Reply:
Oh, nice catch! Will update this. Thanks 😄